core/src/impl/Kokkos_Abort.cpp

```diff
@@ -0,0 +1,47 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#ifndef KOKKOS_IMPL_PUBLIC_INCLUDE
+#define KOKKOS_IMPL_PUBLIC_INCLUDE
+#endif
+
+#include <cstdlib>
+#include <iostream>
+#include <Kokkos_Abort.hpp>
+#include <impl/Kokkos_Error.hpp>
+#include <impl/Kokkos_Stacktrace.hpp>
+
+namespace Kokkos {
+namespace Impl {
+
+inline void traceback_callstack(std::ostream &msg) {
```

Comment:
```suggestion
static void traceback_callstack(std::ostream &msg) {
```
The inline is pointless in the current version.  Either define it the anonymous namespace or add static.

Reply:
Function body inlined into `host_abort`.